### PR TITLE
fix(email-connector): proposition to mitigate the issue of deletion

### PR DESCRIPTION
**Pull request by mathias-vandaele**
## Description

Remove the deleteOnRead parameter as It is always done.

## Related issues

closes #

## Checklist

- [ ] PR has a **milestone** or the `no milestone` label.

### CODE REVIEW

**Comment by ev-codes** [comment]
@mathias-vandaele it looks like this PR needs rebase onto `main`

File: connectors/email/element-templates/email-outbound-connector.json [hunk 1]
```diff
@@ -685,6 +685,7 @@
   }, {
     "id" : "searchStringEmailPop3",
     "label" : "Search criteria",
+    "description" : "Refer to our detailed documentation for full search syntax and examples: [Email Documentation](https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/email/).",
```
Comment (johnBgood) [inline connectors/email/element-templates/email-outbound-connector.json:688]: not sure but I guess links need to be relative(?)

**Review by johnBgood — APPROVED** [review]

**Comment by sbuettner** [comment]
Backport failed for `release/8.6`, because it was unable to cherry-pick the commit(s).

Please cherry-pick the changes locally and resolve any conflicts.
```bash
git fetch origin release/8.6
git worktree add -d .worktree/backport-3381-to-release/8.6 origin/release/8.6
cd .worktree/backport-3381-to-release/8.6
git switch --create backport-3381-to-release/8.6
git cherry-pick -x 87af910451092ef6ad99933a2b081ec677ed18c4
```

**Comment by sbuettner** [comment]
Backport failed for `release/8.6`, because it was unable to cherry-pick the commit(s).

Please cherry-pick the changes locally and resolve any conflicts.
```bash
git fetch origin release/8.6
git worktree add -d .worktree/backport-3381-to-release/8.6 origin/release/8.6
cd .worktree/backport-3381-to-release/8.6
git switch --create backport-3381-to-release/8.6
git cherry-pick -x 87af910451092ef6ad99933a2b081ec677ed18c4
```

**Comment by sbuettner** [comment]
Backport failed for `release/8.6`, because it was unable to cherry-pick the commit(s).

Please cherry-pick the changes locally and resolve any conflicts.
```bash
git fetch origin release/8.6
git worktree add -d .worktree/backport-3381-to-release/8.6 origin/release/8.6
cd .worktree/backport-3381-to-release/8.6
git switch --create backport-3381-to-release/8.6
git cherry-pick -x 87af910451092ef6ad99933a2b081ec677ed18c4
```

**Comment by sbuettner** [comment]
Backport failed for `release/8.6`, because it was unable to cherry-pick the commit(s).

Please cherry-pick the changes locally and resolve any conflicts.
```bash
git fetch origin release/8.6
git worktree add -d .worktree/backport-3381-to-release/8.6 origin/release/8.6
cd .worktree/backport-3381-to-release/8.6
git switch --create backport-3381-to-release/8.6
git cherry-pick -x 87af910451092ef6ad99933a2b081ec677ed18c4
```

**Comment by sbuettner** [comment]
Backport failed for `release/8.6`, because it was unable to cherry-pick the commit(s).

Please cherry-pick the changes locally and resolve any conflicts.
```bash
git fetch origin release/8.6
git worktree add -d .worktree/backport-3381-to-release/8.6 origin/release/8.6
cd .worktree/backport-3381-to-release/8.6
git switch --create backport-3381-to-release/8.6
git cherry-pick -x 87af910451092ef6ad99933a2b081ec677ed18c4
```

**Comment by sbuettner** [comment]
Backport failed for `release/8.6`, because it was unable to cherry-pick the commit(s).

Please cherry-pick the changes locally and resolve any conflicts.
```bash
git fetch origin release/8.6
git worktree add -d .worktree/backport-3381-to-release/8.6 origin/release/8.6
cd .worktree/backport-3381-to-release/8.6
git switch --create backport-3381-to-release/8.6
git cherry-pick -x 87af910451092ef6ad99933a2b081ec677ed18c4
```

**Comment by sbuettner** [comment]
Backport failed for `release/8.6`, because it was unable to cherry-pick the commit(s).

Please cherry-pick the changes locally and resolve any conflicts.
```bash
git fetch origin release/8.6
git worktree add -d .worktree/backport-3381-to-release/8.6 origin/release/8.6
cd .worktree/backport-3381-to-release/8.6
git switch --create backport-3381-to-release/8.6
git cherry-pick -x 87af910451092ef6ad99933a2b081ec677ed18c4
```

**Comment by sbuettner** [comment]
Backport failed for `release/8.6`, because it was unable to cherry-pick the commit(s).

Please cherry-pick the changes locally and resolve any conflicts.
```bash
git fetch origin release/8.6
git worktree add -d .worktree/backport-3381-to-release/8.6 origin/release/8.6
cd .worktree/backport-3381-to-release/8.6
git switch --create backport-3381-to-release/8.6
git cherry-pick -x 87af910451092ef6ad99933a2b081ec677ed18c4
```

**Comment by sbuettner** [comment]
Backport failed for `release/8.6`, because it was unable to cherry-pick the commit(s).

Please cherry-pick the changes locally and resolve any conflicts.
```bash
git fetch origin release/8.6
git worktree add -d .worktree/backport-3381-to-release/8.6 origin/release/8.6
cd .worktree/backport-3381-to-release/8.6
git switch --create backport-3381-to-release/8.6
git cherry-pick -x 87af910451092ef6ad99933a2b081ec677ed18c4
```

**Comment by sbuettner** [comment]
Backport failed for `release/8.6`, because it was unable to cherry-pick the commit(s).

Please cherry-pick the changes locally and resolve any conflicts.
```bash
git fetch origin release/8.6
git worktree add -d .worktree/backport-3381-to-release/8.6 origin/release/8.6
cd .worktree/backport-3381-to-release/8.6
git switch --create backport-3381-to-release/8.6
git cherry-pick -x 87af910451092ef6ad99933a2b081ec677ed18c4
```

**Comment by sbuettner** [comment]
Backport failed for `release/8.6`, because it was unable to cherry-pick the commit(s).

Please cherry-pick the changes locally and resolve any conflicts.
```bash
git fetch origin release/8.6
git worktree add -d .worktree/backport-3381-to-release/8.6 origin/release/8.6
cd .worktree/backport-3381-to-release/8.6
git switch --create backport-3381-to-release/8.6
git cherry-pick -x 87af910451092ef6ad99933a2b081ec677ed18c4
```

**Comment by sbuettner** [comment]
Backport failed for `release/8.6`, because it was unable to cherry-pick the commit(s).

Please cherry-pick the changes locally and resolve any conflicts.
```bash
git fetch origin release/8.6
git worktree add -d .worktree/backport-3381-to-release/8.6 origin/release/8.6
cd .worktree/backport-3381-to-release/8.6
git switch --create backport-3381-to-release/8.6
git cherry-pick -x 87af910451092ef6ad99933a2b081ec677ed18c4
```

**Comment by sbuettner** [comment]
Backport failed for `release/8.6`, because it was unable to cherry-pick the commit(s).

Please cherry-pick the changes locally and resolve any conflicts.
```bash
git fetch origin release/8.6
git worktree add -d .worktree/backport-3381-to-release/8.6 origin/release/8.6
cd .worktree/backport-3381-to-release/8.6
git switch --create backport-3381-to-release/8.6
git cherry-pick -x 87af910451092ef6ad99933a2b081ec677ed18c4
```

**Comment by sbuettner** [comment]
Backport failed for `release/8.6`, because it was unable to cherry-pick the commit(s).

Please cherry-pick the changes locally and resolve any conflicts.
```bash
git fetch origin release/8.6
git worktree add -d .worktree/backport-3381-to-release/8.6 origin/release/8.6
cd .worktree/backport-3381-to-release/8.6
git switch --create backport-3381-to-release/8.6
git cherry-pick -x 87af910451092ef6ad99933a2b081ec677ed18c4
```

**Comment by sbuettner** [comment]
Backport failed for `release/8.6`, because it was unable to cherry-pick the commit(s).

Please cherry-pick the changes locally and resolve any conflicts.
```bash
git fetch origin release/8.6
git worktree add -d .worktree/backport-3381-to-release/8.6 origin/release/8.6
cd .worktree/backport-3381-to-release/8.6
git switch --create backport-3381-to-release/8.6
git cherry-pick -x 87af910451092ef6ad99933a2b081ec677ed18c4
```

**Pull request merged**